packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/services/DownloadService.java

```diff
@@ -55,7 +55,7 @@ public static Boolean IsPdf(final byte[] data) {
 
 		// for pdf version 1.3
 		// check if data ends with %%EOF
-		if (data[5] == 0x31 && data[6] == 0x2E && data[7] == 0x33) {
+		if (data[4] == 0x31 && data[5] == 0x2E && data[6] == 0x33) {
```

Comment:
apparently this is correct for checking for pdf, this was incorrect before and was flooding the document-assets with large text objects

Reply:
wow :o